playground.sql

```diff
@@ -0,0 +1,5 @@
+\c nc_games
+
+\dt
+
+SELECT * FROM categories;
```

Comment:
I would reccommend git ignoring these playground files as they aren't a part of the working code.